client/my-sites/stats/most-popular/style.scss

```diff
@@ -79,6 +79,8 @@
 	@include breakpoint( ">660px" ) {
 		padding: 13px 48px;
 		font-size: inherit;
+		display:inline-flex;
```

Comment:
A total nit pic but could you add a space after `:`? 

(Linters job to warn about that but seems like it's not configured to do so 🤔 ).